common/w_wad.h

```diff
@@ -88,12 +88,34 @@ typedef enum {
 	ns_colormaps,
 } namespace_t;
 
+struct lumpHandle_t
+{
+	size_t id;
+	lumpHandle_t() : id(0)
```

Comment:
Consider making the constructor explicit

Reply:
Disregard. The constructor is a default constructor. There is no constructor that takes a parameter.